src/app/issuer/components/issuer-create/issuer-create.component.ts

```diff
@@ -80,13 +89,42 @@ export class IssuerCreateComponent extends BaseAuthenticatedRoutableComponent im
 		super.ngOnInit();
 	}
 
+	ngAfterViewInit(): void {
+		this.captchaService.getCaptcha().then((captcha) => {
```

Comment:
The exact same code snippet is repeated in `signup.component.ts`, 
if possible? add it to `app -> common -> util`